### PR TITLE
[Modules] Fix voip_play

### PR DESCRIPTION
**Pull request by gpotter2**
Made the voip_play functions work again (fixed bugs) + adapted them to work under windows

It theorically work under linux, but i couldn't test it.

Fixes:
- Remove FIFO usage: really outdated, slow, and not working under windows
- Several fixes on the functions (IP packet field detection, RTP not properly used)

### CODE REVIEW

**Comment by gpotter2** [comment]
Fixes https://github.com/secdev/scapy/issues/432

File: scapy/modules/voip.py [hunk 1]
```diff
@@ -7,94 +7,78 @@
...
+    except WindowsError:
+        raise OSError("Sox must be installed to play VoIP packets")
+    finally:
+        if p_test:
+            p_test.terminate()
+else:
+    from fcntl import fcntl
 
 def merge(x,y,sample_size=2):
-    if len(x) > len(y):
```
Comment (gpotter2) [inline scapy/modules/voip.py]: That was the cause of horrible sound: sox was already handling the empty bytes

**Comment by p-l-** [comment]
I'd be happy to test that under Linux if you can share a test pcap file?

**Comment by gpotter2** [comment]
Here:
http://www.filedropper.com/voip
Test it with
`>>> voip_play("64.2.142.189", list=sniff(offline="voip.pcap"))`

**Comment by p-l-** [comment]
When you install `sox` on windows, does it also install a `play` binary?

**Comment by gpotter2** [comment]
No it doesn't.

**Comment by gpotter2** [comment]
@p-l- Any news on this ?

**Comment by gpotter2** [comment]
I might haven found a way to remove distorsion. Please do not merge yet

**Comment by gpotter2** [comment]
This PR will now work without https://github.com/secdev/scapy/pull/503 on windows. (to avoid conflicts)
Please merge https://github.com/secdev/scapy/pull/503 before

**Comment by p-l-** [comment]
@gpotter2 now #503 has been merged, maybe you could rebase against current master?

**Comment by gpotter2** [comment]
@p-l- Done

**Comment by gpotter2** [comment]
Please restart the failing Travis-ci tests

**Pull request merged**